src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResourceIntTest.kt

```diff
@@ -748,8 +759,8 @@ class VisitResourceIntTest : IntegrationTestBase() {
     }
   }
 
-  private fun createVisit(startDateTime: String, endTime: String, room: String, openClosedStatus: String) =
-    webTestClient.post().uri("/prisoners/$offenderNo/visits")
+  private fun createVisit(startDateTime: String, endTime: String, room: String, openClosedStatus: String, offenderNoForVisit: String = offenderNo) =
+    webTestClient.post().uri("/prisoners/$offenderNoForVisit/visits")
```

Comment:
needed a different prisoner since this test accidentally created 2  identical visits